subsys/net/ip/rpl.c

```diff
@@ -649,13 +649,20 @@ static void dio_timer(struct k_work *work)
 			struct in6_addr *addr =
 				net_if_ipv6_get_ll(rpl_default_iface,
 						   NET_ADDR_PREFERRED);
+			if (addr) {
+				net_rpl_dio_send(rpl_default_iface, instance, addr,
+					         NULL);
+				#if defined(CONFIG_NET_STATISTICS_RPL)
```

Comment:
We typically do not use indented pre-processor code, so #if should start from first column.

Reply:
Please start the #if from the column 0